### PR TITLE
Exploratory analysis nov20

### PR DESCRIPTION
**Pull request by luizaandrade**

### CODE REVIEW

File: Presentations/04-descriptive-analytics/04-descriptive-analytics.Rmd [hunk 1]
```diff
@@ -0,0 +1,919 @@
...
+---
+
+# Customizing *skimr*
+
+.small[
+```{r}
+census %>% 
+  summary_stats() %>%
+  yank("numeric") %>% # keep only numeric variables on the table
+  select(-n_missing, -complete_rate) # remove daulft statistics
```
Comment (luisesanmartin) [inline Presentations/04-descriptive-analytics/04-descriptive-analytics.Rmd:479]:
```suggestion
  select(-n_missing, -complete_rate) # remove default statistics
```

File: Presentations/04-descriptive-analytics/04-descriptive-analytics.Rmd [hunk 2]
```diff
@@ -0,0 +1,919 @@
...
+   summary_stats_table %>%
+   rename(name = skim_variable) %>% # Rename var with var names so we can merge the datasets
+   left_join(census_dictionary) %>% # Merge to variable labels
+   select(-name) %>% # Keep only variable labels instead of names
+   as_hux # Convert it into a huxtable object
+
+
+summary_stats_table <-
+   summary_stats_table %>% 
+   relocate(Variable) %>%  # Make veriable labels the first column
```
Comment (luisesanmartin) [inline Presentations/04-descriptive-analytics/04-descriptive-analytics.Rmd:546]:
```suggestion
   relocate(Variable) %>%  # Make variable labels the first column
```

File: Presentations/04-descriptive-analytics/04-descriptive-analytics.Rmd [hunk 3]
```diff
@@ -0,0 +1,919 @@
...
+
+- **formula:**	an object of class "formula" containing a symbolic description of the model
+- **data:** a data frame containing the variables indicated in the formula
+- *subset:* an optional vector specifying a subset of observations to be used in the regression
+- *weights:* an optional vector of weights to be used in the regression
+]
+
+Formulas can take three specifications:
+- `y ~ x1 + x2` regresses variable `y` on covariates `x1` and `x2`
+- `y ~ x1:x2` regresses variable `y` the interaction of covariates `x1` and `x2`
```
Comment (luisesanmartin) [inline Presentations/04-descriptive-analytics/04-descriptive-analytics.Rmd:707]:
```suggestion
- `y ~ x1:x2` regresses variable `y` on the interaction of covariates `x1` and `x2`
```

File: Presentations/04-descriptive-analytics/04-descriptive-analytics.Rmd [hunk 4]
```diff
@@ -0,0 +1,919 @@
...
+- `y ~ x1*x2` is equivalent to `y ~ x1 + x2 + x1:x2`
+
+---
+
+# Running regressions
+
+.exercise[
+### Exercise `r fa("keyboard")`
+Using the `census` data, run a regression of population, urban population and number of marriages.
```
Comment (luisesanmartin) [inline Presentations/04-descriptive-analytics/04-descriptive-analytics.Rmd:716]: the outputs shown later for the solution of this exercise show a regression of divorces on population, urban population and number of marriages. Shouldn't the prompt for the exercise be: _"run a regression of divorces on population, urban population and number of marriages"_?

Comment (luisesanmartin) [reply Presentations/04-descriptive-analytics/04-descriptive-analytics.Rmd:716]: this also applies for the next two slides and the next exercise (FE regression)

**Review by luisesanmartin — APPROVED** [review]
A couple of comments correcting typos and one suggestion to change the prompt of two exercises

File: Exercise solutions/Descriptive Analysis.R [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+dataWorkFolder    <- file.path(projectFolder,"DataWork")
+Data              <- file.path(dataWorkFolder,"DataSets")
+finalData         <- file.path(Data,"Final")
+rawOutput         <- file.path(dataWorkFolder,"Output","Raw")
+
+# Load packages
+packages <- c("tidyverse", 
+              "skimr", 
+              "huxtable")
+
```
Comment (luizaandrade) [inline Exercise solutions/Descriptive Analysis.R:18]:
```suggestion
install.packages("pacman") # this package tests if a package is installed before loading, and installs it if it is not
```

**Pull request merged**